os-config.tpl.yml

```diff
@@ -105,7 +105,7 @@ rancher:
         io.rancher.os.detach: "false"
         io.rancher.os.reloadconfig: "true"
         io.rancher.os.scope: system
-        io.rancher.os.after: cloud-init-pre, wait-for-network
+        io.rancher.os.after: cloud-init-pre, wait-for-network, ntp
```

Comment:
There's no guarantee, of course, but a pretty good chance. Otherwise ubuntu-console or other external services that are loaded via HTTPS just can't load on boot. 

We could make wait-for-network depend on ntp if that looks better.

Another approach could be to ditch HTTPS and use hash-sums (cached in the config), though this would make configuration a bit bloated and prone to errors. But that's probably a topic for another discussion.